### PR TITLE
[MM-60830] Replace FormattedMarkdownMessage in 'webapp/channels/src/components/user_list_row/user_list_row.tsx' with FormattedMessage

### PR DESCRIPTION
**Pull request by TheInvincibleRalph**
#### Ticket Link
Fixes https://github.com/mattermost/mattermost/issues/28496

```release-note
NONE
```

### CODE REVIEW

**Comment by devinbinnie** [comment]
@TheInvincibleRalph Can you check on the failing lint check?

**Comment by TheInvincibleRalph** [comment]
> @TheInvincibleRalph Can you check on the failing lint check?

Done @devinbinnie. Thank you!

**Review by devinbinnie — APPROVED** [review]
LGTM! Thanks @TheInvincibleRalph!

File: webapp/channels/src/components/user_list_row/user_list_row.tsx [hunk 1]
```diff
@@ -79,11 +79,12 @@ const UserListRow = ({user, status, extraInfo = [], actions = [], actionProps, a
     let statusProp: string | undefined;
     if (extraInfo && extraInfo.length > 0) {
         emailProp = (
-            <FormattedMarkdownMessage
-                id='admin.user_item.emailTitle'
-                defaultMessage='**Email:** {email}'
+            <FormattedMessage
+                id='admin.user_item.emailTitle_admin'
```
Comment (larkox) [inline webapp/channels/src/components/user_list_row/user_list_row.tsx:83]: Why the admin suffix?

**Review by larkox — CHANGES_REQUESTED** [review]
Please check the linting errors. Apart of that just one minor question.

File: webapp/channels/src/components/user_list_row/user_list_row.tsx (hunk 1 above)
Comment (TheInvincibleRalph) [reply webapp/channels/src/components/user_list_row/user_list_row.tsx:83]: I did that based on the instruction on the issue:

```...change the message ID after the update as the current extraction tool does not automatically detect changes in the messages.```

**Comment by TheInvincibleRalph** [comment]
> Please check the linting errors. Apart of that just one minor question.

I will look into that now. Thank you!

File: webapp/channels/src/components/user_list_row/user_list_row.tsx (hunk 1 above)
Comment (TheInvincibleRalph) [reply webapp/channels/src/components/user_list_row/user_list_row.tsx:83]: Your ID suggestion looks fine. I'll make the changes. Thank you!

Comment (TheInvincibleRalph) [reply webapp/channels/src/components/user_list_row/user_list_row.tsx]: I have made the changes, @larkox.

**Comment by TheInvincibleRalph** [comment]
Hi, @M-ZubairAhmed. The CI-lint step should pass now.

**Review by larkox — APPROVED** [review]

**Comment by TheInvincibleRalph** [comment]
Everything looks fine from my end 💯

**Review by yasserfaraazkhan — APPROVED** [review]

**Pull request merged**